counsel.el

```diff
@@ -4527,13 +4532,18 @@ If there is no such buffer, start a new `shell' with NAME."
   "Face used by `counsel-firefox-bookmarks' for tags."
   :group 'ivy-faces)
 
+(defface counsel-firefox-bookmarks-location
+  '((t :inherit link))
+  "Face used by `counsel-firefox-bookmarks' for locations."
```

Comment:
"Location" can mean several things, please specify that you are talking about URLs instead.

Reply:
Firefox's bookmark manager use the "location" term, so i prefer to use the same one

Reply:
But the user does not know this. If you insist on using the term "location" then you should at least add another sentence explaining what it is.